### PR TITLE
Add Modal Component

### PR DESCRIPTION
**Pull request by varun13169**
Add Modal Component

### CODE REVIEW

**Review by zoef-yt — APPROVED** [review]
Clean code. Looks good to me. Good to merge.

File: components/modal/modalPage.css [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    overflow: auto;
+}
+
+.is-blurred {
+    -webkit-filter: blur(7px);
+    -moz-filter: blur(7px);
+    -o-filter: blur(7px);
+    -ms-filter: blur(7px);
+    filter: blur(7px);
+}
```
Comment (iRohitGaur) [inline components/modal/modalPage.css:65]: the blur looks nice

File: components/modal/modalPage.html [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+            <div id="container__code-preview" class="container__code-preview">
+                <script type="text/plain" class="language-markup">
+                    <div class="dui-modal dui-util-disp-none dui-util-gry-shdw">
+                        <div class="dui-modal__info-card dui-util-spc-pad-s">
+                            <p class="dui-modal__close">X</p>
+                            <h2 class="dui-modal__title dui-util-txt-align-cent">Subscribe to our Newsletter !!!</h2>
+                            <button
+                                class="dui-btn dui-btn--primary dui-util-bdr-radi-5px-s dui-util-txt-sm dui-util-spc-pad-0_8rem-xs reset-button-inherit-parent">
+                                SUNSCRIBE
```
Comment (iRohitGaur) [inline components/modal/modalPage.html:160]: SUBSCRIBE*

**Review by iRohitGaur — APPROVED** [review]
Nice work! Good to merge 👍🏻

**Review by archit7448 — APPROVED** [review]
Good to merge .

File: components/modal/modalPage.css (hunk 1 above)
Comment (varun13169) [reply components/modal/modalPage.css:65]: Thanks.

File: components/modal/modalPage.html (hunk 2 above)
Comment (varun13169) [reply components/modal/modalPage.html:160]: Done, thanks for pointing it out.

**Pull request merged**